Lib/webbrowser.py

```diff
@@ -475,35 +425,25 @@ def register_X_browsers():
     if "KDE_FULL_SESSION" in os.environ and shutil.which("kfmclient"):
         register("kfmclient", Konqueror, Konqueror("kfmclient"))
 
+    # Common symbolic link for the default X11 browser
     if shutil.which("x-www-browser"):
         register("x-www-browser", None, BackgroundBrowser("x-www-browser"))
```

Comment:
Good spot!

I came across this issue - unsure if this could be a fix? https://github.com/python/cpython/issues/101893

I saw your edit :) but is it worth adding sensible-browser for good measure? What are your thoughts?

Reply:
See my message: no need to add sensible-browser at this point.